basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -149,9 +175,8 @@ ERROR: unknown-chloe-tag tag ;
 
 : compile-prologue ( xml -- )
     [
-        [ prolog>> [ write-xml ] [code-with] ]
-        [ before>> compile-chunk ]
-        bi
+        "<!DOCTYPE html>\n" [write]
```

Comment:
So no more xhtml? I suppose that's because it's all deprecated in favor of the "future is now"?